hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/helpers/KafkaSourceUtil.java

```diff
@@ -47,4 +49,36 @@ public static void configureSchemaDeserializer(SchemaProvider schemaProvider, Ty
         : StringUtils.concatenateWithThreshold(String.format("%s_", groupId), schemaHash, GROUP_ID_MAX_BYTES_LENGTH);
     props.put(NATIVE_KAFKA_CONSUMER_GROUP_ID, updatedConsumerGroup);
   }
+
+  /**
+   * Utility method that removes configs with keys that match (start with) any one of the prefixes.
+   *
+   * @param kafkaParams The incoming kafka params
+   * @param commaSeparatedPrefixes all configs with keys starting with any one of these comma-separated prefixes will be ignored.
+   * @return a new set of kafkaParams with the configs matching the prefixes removed.
+   */
+  public static Map<String, Object> filterKafkaParameters(Map<String, Object> kafkaParams, String commaSeparatedPrefixes) {
+    if (commaSeparatedPrefixes.isEmpty()) {
+      return kafkaParams;
+    }
+
+    String[] prefixes = commaSeparatedPrefixes.split(";");
```

Comment:
documentation says, comma separated list of prefixes and here we are splitting by `;` ?

Reply:
sorry the documentation was not correct, fixed it